src/providers/osso.js

```diff
@@ -0,0 +1,20 @@
+export default (options) => {
+  return {
+    id: 'osso',
+    name: 'SAML SSO',
+    type: 'oauth',
+    version: '2.0',
+    params: { grant_type: 'authorization_code' },
+    accessTokenUrl: `https://${options.domain}/oauth/token`,
+    authorizationUrl: `https://${options.domain}/oauth/authorize?response_type=code`,
+    profileUrl: `https://${options.domain}/oauth/me`,
+    profile: (profile) => {
+      return {
+        id: profile.id,
+        name: profile.name || profile.email,
```

Comment:
SAML spec is a little odd wrt NAMEID format things. Osso's docs don't instruct IDP admins to include a "name" attribute, but we do sometimes get them back.

I'm thinking here that if the database or user model depends on there being a name, we should at least fall back to email, which is guaranteed to be available.